packages/components/src/box-control/all-input-control.js

```diff
@@ -14,6 +14,7 @@ export default function AllInputControl( {
 	onHoverOn = noop,
 	onHoverOff = noop,
 	values,
+	sides,
```

Comment:
Maybe it  should be just an array instead of a object: `[ 'top', 'bottom' ]` would show just top and bottom.

Reply:
The switch to using an array instead of object has been made.